.github/workflows/branch.yml

```diff
@@ -0,0 +1,135 @@
+name: Kirby branch CI
+on:
+  push:
+    branches-ignore: [ master ]
+defaults:
+  run:
+    shell: bash
+
+jobs:
+    lint_and_unittest:
+      runs-on: ubuntu-latest
+      strategy:
+       matrix:
+          node-version: [10.16]
+      steps:
+        - uses: actions/checkout@v2
+        - name: Use Node.js ${{ matrix.node-version }}
+          uses: actions/setup-node@v1
+          with:
+            node-version: ${{ matrix.node-version }}
+        - name: Install six
+          run: pip install six
+        - name: run npm ci
+          run: npm ci
+        - name: run npm lint
+          run: npm run lint
+        - name: run npm extract-scss-variables
+          run: npm run extract-scss-variables
+        - name: run npm test ci
+          run: npm run test:ci
+        - name: run npm coverage-report
+          run: npm run coverage-report || true
+
+    build_kirby:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          node-version: [10.16]
+      steps:
+        - uses: actions/checkout@v2
+        - name: Use Node.js ${{ matrix.node-version }}
+          uses: actions/setup-node@v1
+          with:
+            node-version: ${{ matrix.node-version }}
+        - name: install six
+          run: pip install six
+        - name: run npm ci
+          run: npm ci
+        - name: run npm dist:designsystem
+          run: npm run dist:designsystem
+
```

Comment:
This step seems obsolete ? The deploy step builds and deploys the cookbook, the publish step builds and publishes the designsystem lib/package...
```suggestion
```

Reply:
Don't we need to see if the branch builds before being able to merge?
A future development of this could be to cache the build from this step and use it in the deploy step.